### PR TITLE
Add service comparison component

### PR DESCRIPTION
**Pull request by bytheway875**
Screen Shot 2022-02-18 at 4 52 40 PM
Screen Shot 2022-02-18 at 4 52 51 PM

Route is added as temporary, demo/development only route so that we can get translation strings into transifex

### CODE REVIEW

File: app/views/shared/_service_comparison.html.erb [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+            </div>
+          </div>
+        </div>
+        <div class="service-set service-set--mobile-only">
+          <div class="service-set__header">
+          </div>
+
+          <div class="service-set__services">
+            <div class="service full-service">
+              <strong><%= "GYR" %></strong>
```
Comment (alaroia-cfa) [inline app/views/shared/_service_comparison.html.erb:74]: Should these be I18n'd? Not a blocker.

File: app/views/shared/_service_comparison.html.erb [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+            <div class="service-set__title">
+              <%= t("views.shared.service_comparison.filing_years.title") %>
+            </div>
+          </div>
+          <div class="service-set__services">
+            <div class="service full-service">
+              <%= TaxReturn.filing_years.first %>-<%= TaxReturn.filing_years.last %>
+            </div>
+            <div class="service diy">
+              <%= TaxReturn.current_tax_year %>
```
Comment (alaroia-cfa) [inline app/views/shared/_service_comparison.html.erb:115]: Genius

File: app/views/shared/_service_comparison.html.erb [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+            <div class="service-set__title service-set__title--normal">
+              <%= t("views.shared.service_comparison.tax_documents.title") %>
+            </div>
+            <div class="service-set__subtitle">
+              <%= t("views.shared.service_comparison.tax_documents.subtitle") %>
+            </div>
+          </div>
+          <div class="service-set__services">
+            <div class="service full-service">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
```
Comment (alaroia-cfa) [inline app/views/shared/_service_comparison.html.erb:185]: Wanna i18n the "yes"? Not a blocker.

File: app/views/shared/_service_comparison.html.erb [hunk 4]
```diff
@@ -0,0 +1,360 @@
...
+        </div>
+        <div class="service-set">
+          <div class="service-set__header">
+            <div class="service-set__title">
+              <%= t("views.shared.service_comparison.credits_and_payments") %>
+            </div>
+          </div>
+          <div class="service-set__services">
+            <div class="service full-service">
+              <%= image_tag "icons/lots-of-cash.svg", alt: "all tax credits" %>
```
Comment (alaroia-cfa) [inline app/views/shared/_service_comparison.html.erb:202]: Wanna i18n the `alt=`, here and below? Not a blocker.

File: app/views/shared/_service_comparison.html.erb [hunk 5]
```diff
@@ -0,0 +1,360 @@
...
+          <div class="service-set__header">
+            <div class="service-set__title service-set__title">
+              <%= t("views.shared.service_comparison.additional_benefits") %>
+            </div>
+          </div>
+          <div class="service-set__services service-set--mobile-excluded">
+            <div class="service full-service">
+            </div>
+            <div class="service diy">
+
```
Comment (alaroia-cfa) [inline app/views/shared/_service_comparison.html.erb:292]: This blank line in the div freaks me out and makes me wonder if it's meaningful (`white-space: pre-wrap` or something). I'm going to assume it's not meaningful. Not a blocker.

File: app/views/shared/_service_comparison.html.erb [hunk 6]
```diff
@@ -0,0 +1,360 @@
...
+            </div>
+          </div>
+          <div class="service-set__services">
+            <div class="service full-service">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service diy">
+
+            </div>
+            <div class="service express">
```
Comment (alaroia-cfa) [inline app/views/shared/_service_comparison.html.erb:348]: These might be tabular data, which might be a good use of tables, but it's really fine the way it is, too. When reading the HTML, it took me a while to realize that these divs were really table cells (and empty table cells at that).

File: app/assets/stylesheets/components/_service-comparison.scss [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+    &__subtitle {
+      font-style: italic;
+      display: flex;
+      font-size: 1.4rem;
+      line-height: 1.9rem;
+      padding-bottom: .5rem;
+      @media screen and (min-width: $site-max-up) {
+        text-align: right;
+      }
+      &--fixie {
```
Comment (alaroia-cfa) [inline app/assets/stylesheets/components/_service-comparison.scss:88]: fixie is a fun name, but I wasn't really sure what it meant (I was expecting a [clearfix](https://css-tricks.com/snippets/css/clear-fix/)).

**Review by alaroia-cfa — APPROVED** [review]
I left some comments. I found no blockers. Cheerio.

File: app/views/shared/_service_comparison.html.erb (hunk 1 above)
Comment (bytheway875) [reply app/views/shared/_service_comparison.html.erb:74]: oh yes, probably.

**Pull request merged**